src/temp-types-based-on-js-source.ts

```diff
@@ -74,7 +75,10 @@ export interface ParserOptions {
   tokens: boolean;
   comment: boolean;
   jsx: boolean;
+  ecmaFeatures: { jsx: boolean };
```

Comment:
Again, no need for this and jsx: boolean above I don't think

Reply:
Removed.